engine/installation/linux/debian.md

```diff
@@ -105,10 +99,10 @@ Docker from the repository.
       uid                  Docker Release Tool (releasedocker) <docker@docker.com>
     ```
 
-3.  Use the following command to set up the **stable** repository. To also
-    enable the **testing** repository, add the words `testing` after `main` on
-    the last line.
-    **Do not use these unstable repositories on production systems or for non-testing workloads.**
+3.  Use the following command to set up the **stable** repository.
+
+    > **Note**: The `lsb_release -cs` sub-command below returns the name of your
+    > Debian distribution, such as `jessie`. For Raspbian, change it to `raspbian-$(lsb_release -cs)`.
```

Comment:
I did have them split up but undid it in response to https://github.com/docker/docker.github.io/pull/1560#discussion_r100176231. @friism WDYT?